### ISSUE TITLE
Add a light mode UI option

### ISSUE THREAD

**Issue by Mywk** [safing/portmaster-ui#422]
**What would you like to add or change?**:
The ability to have a light mode UI instead of dark mode only.

**Why do you and others need this?**:
The purpose of this feature is to improve the professional appearance of the UI, particularly in corporate environments.

As of right now the UI only supports a dark mode which may not be suitable for all users, especially those who prefer a lighter and more traditional appearance. While this is an extremely low priority suggestion it would be nice to see it implemented in the future. I also think this feature would be potentially more appealing to a wider range of users.

**Comment by MGvK** [comment]
Also improves readability when working in sunlight, esp. with notebook outside.

**Comment by dhaavi** [comment]
Light mode is planned, but as you said it has an "extremely low priority".

**Comment by p0358** [comment]
+1, I would like to point out that light mode themes are much easier to read for people with astigmatism, while dark mode makes it harder for the eyes to focus on the text (which is the case for all eyes, with astigmatism the effect is just amplified), so it is kinda sad that it's considered an "extremely" low priority feature request for over two years

I think a group of people could appreciate dedicating some resources to the theming for the benefit of accessibility

**Comment by osmin1** [comment]
I don't envy the developers. After 30 minutes my eyes just cannot take it anymore. But they have to work on it regardless. I think there is a reason why Windows departed from "dark theme" native to DOS...
Another thing that makes it very unfriendly to use are scrollbars. Does anyone expect it to be used on a phone? If not, there is no reason to use phone-like scrollbars. Just give us normally sized scrollbars, preferably with steppers.

**Comment by swanknight** [comment]
I despise dark mode. It might be good for your battery (OLED only), but it's terrible for the eyes.
I discovered Portmaster, and after installing it, I was baffled that there was no way to switch to a "light" UI.
After coming across this GitHub issue, I immediately uninstalled it.

**Comment by p0358** [comment]
Yeah I ended up uninstalling fairly quickly too, no point in torturing myself like that tbh

**Comment by OfficialEsco** [comment]
Linking https://github.com/safing/portmaster-ui/issues/169 as that is kind of the original request

**Comment by p0358** [comment]
I wonder if removal of backlog label means they're currently working on this now, or not planning to anymore xd

**Comment by Mywk** [comment]
Ping

**Comment by Mywk** [comment]
Ping! Bad bot!

**Comment by p0358** [comment]
> I don't envy the developers. After 30 minutes my eyes just cannot take it anymore. But they have to work on it regardless.

I got a new IPS screen with high brightness and I think I understand now. They just never tried the app on anything but the highest-end displays. It's on the edge of being bearable enough if I set the brightness to 100% (on a monitor that has 350 cd/m2, ~100 higher than usual). Have anything weaker like default brightness, and it becomes questionable. Have a TN panel and it's straight up unusable and immediate uninstall.

So it's not only hard on people with poor sight, but also doubles down on poor people I guess lol
